django_react_proj/users/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.2 on 2021-04-18 14:56
+# Generated by Django 3.1.7 on 2021-04-18 20:14
```

Comment:
This looks like a problem with versioning, this might not be case right now but it's a good practice to keep hardcoded versions of packages in your `requirements.txt` you use to avoid compatibility differences.